docs/running_locally.md

```diff
@@ -93,7 +93,7 @@ The lottery runs in two steps:
 - Fetch all the registered users
 
 ```
-python borderland_tool -t localhost:8000/token -s localhost:8000 -o ORGANISATION -e EVENT lottery -q QUOTA -f registrations fetch
+python borderland_tool -t localhost:8000/token -s localhost:8000 -o ORGANISATION -e EVENT lottery -q QUOTA -f registrations.csv fetch
```

Comment:
The default file is "lottery.csv" ... we could perhaps change the command line default value as well, so newcomers get an easy experience executing the command.